File: Daily Challenge/Python/Flowchart Logic 062 12.12.2025.py
a, b = map(int, input().split())

while a > 0 and b > 0:
    print(a, end=' ')
    print(b)

    a = a - 1
    b = b - 1

    if a % 10 == 0:
        temp = a
        a = b
        b = temp

File: Daily Challenge/C++/Flowchart Logic 062 12.12.2025.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;
    cin >> a >> b;

    while (a > 0 && b > 0) {
        cout << a << ' ';
        cout << b << '
';

        a = a - 1;
        b = b - 1;

        if (a % 10 == 0) {
            int temp = a;
            a = b;
            b = temp;
        }
    }
    return 0;
}